src/static/riot/queues/list.tag

```diff
@@ -0,0 +1,282 @@
+                            <div class="menu">
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </form>
+        </div>
+        <div class="actions">
+            <div class="ui primary button" onclick="{ handle_queue }">Submit</div>
+            <div class="ui basic red cancel button" onclick="{ close_modal }">Cancel</div>
+        </div>
+    </div>
+
+    <script>
+        var self = this
+        self.queues = []
+        self.selected_queue = {}
+        self.page = 1
+
+        self.one("mount", function () {
+            self.update_queues()
+            $(".ui.checkbox", self.root).checkbox()
+            $('.collab-search')
+                .dropdown({
+                    apiSettings: {
+                        url: `${URLS.API}user_lookup/?q={query}`,
+                    },
+                    clearable: true,
+                    preserveHTML: false,
+                    fields: {
+                        remoteValues: 'results',
+                        name: 'username',
+                        value: 'id',
+                    },
+                    cache: false,
+                    maxResults: 5,
+                })
+            ;
+        })
+
+        self.switch_to_form = () => {
+            window.location = '/queues/form/'
+        }
+
+        self.update_queues = function (filters) {
+            filters = filters || {}
+            let show_public_queues = $(self.refs.public).prop('checked')
+            if (show_public_queues) {
+                filters.public = true
+            }
+            CODALAB.api.get_queues(filters)
+                .done(function (data) {
+                    self.queues = _.orderBy(data.results, queue => !queue.is_owner);
+                    self.pagination = {
+                        "count": data.count,
+                        "next": data.next,
+                        "previous": data.previous
+                    }
+                    self.update()
+                })
+                .fail(function (response) {
+                    toastr.error("Could not load tasks")
+                })
+        }
+
+        /*---------------------------------------------------------------------
+        Modal Methods
+        ----------------------------------------------------------------------*/
+        self.show_modal = (queue) => {
+            if (queue !== undefined && queue !== null) {
+                self.set_selected_queue(queue)
+            }
+            $(self.refs.modal).modal('show')
+        }
+
+        self.close_modal = () => {
+            $(self.refs.modal).modal('hide')
+            self.clear_form()
+        }
+
+        self.clear_form = () => {
+            $('.collab-search').dropdown('clear')
+            self.refs.queue_name.value = ''
+            self.selected_queue = {}
+            self.refs.queue_public.checked = false
```

Comment:
does this work? you don't have to do `$(self.refs.).prop('checked', false)`?

Reply:
I believe this does work

Reply:
I did verify this is working as intended.